### PR TITLE
[ThemeBundle] Fix risky tests

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

File: src/Sylius/Bundle/ThemeBundle/phpunit.xml.dist [hunk 1]
```diff
@@ -6,6 +6,8 @@
 >
     <php>
         <server name="KERNEL_CLASS" value="AppKernel" />
+
+        <server name="SHELL_VERBOSITY" value="-1" />
```
Comment (Zales0123) [inline src/Sylius/Bundle/ThemeBundle/phpunit.xml.dist]: why changing the verbosity?

Comment (pamil) [reply src/Sylius/Bundle/ThemeBundle/phpunit.xml.dist]: Without this change, it is writing a lot of logs to console output when executing tests. It's also the standard Symfony setting: https://github.com/symfony/recipes/blob/master/phpunit/phpunit/4.7/phpunit.xml.dist#L16

Comment (Zales0123) [reply src/Sylius/Bundle/ThemeBundle/phpunit.xml.dist]: Great! 🎆

**Comment by Zales0123** [comment]
Nice job, Kamil! 🍉

**Pull request merged**